main.py

```diff
@@ -12,9 +15,34 @@
 
 graphs = load_graph_list(filename)
 
-os.makedirs(os.path.dirname('output_files/main/'), exist_ok=True)
+# Test output csv creation
+csv_filepath = create_csv_file('main')
+# Write first row with column names
+write_csv_line(csv_filepath, ['file', 'i', 'time (s)', 'result'])
 
+# Test result accumulators
+end_time_sum = 0
+test_result_passed = True
+
+# main test loop
 for i in range(0, len(graphs)):
     outputfilename = 'output_files/main/' + file + '_' + str(i) + '.dot'
+    os.makedirs(os.path.dirname(outputfilename), exist_ok=True)
+
+    start_time: float = time.time()
```

Comment:
Is het nodig om het type te noemen? Ik dacht dat time() van zichzelf al gewoon een float was. Gewoon op onnodige code uit het programma te houden